server/js/sockets.js

```diff
@@ -91,6 +91,7 @@ sockets.init = function socketInit(server) {
   mqttClient.subscribe('power_model/recommended_SP');
   mqttClient.subscribe('power_model/plan_generated');
   mqttClient.subscribe('camera/push_overlays');
+  // camera recording status subscription occurs when 'camera-recording-init' is received
```

Comment:
My own nit: Comments starting with capital letters. Not sure why it wasn't picked up by eslint? https://eslint.org/docs/rules/capitalized-comments

Reply:
Done
130e0c0

Not sure why it wasn't picked up either